src/qt/intro.cpp

```diff
@@ -349,3 +332,39 @@ QString Intro::getPathToCheck()
     mutex.unlock();
     return retval;
 }
+
+void Intro::setSizeWarningLabel(bool isPruning, uint64_t blockchain_size, uint64_t chain_state_size)
```

Comment:
Why have the 2 last arguments if they are always the same?

Reply:
What do you mean?
IIRC the blockchain size and chainstate size are two different things

Reply:
Instead of arguments, you have access to m_blockchain_size and the other.

Reply:
Are you sure?
IIRC it was only available in the contructor but I could be wrong, thats why I used this method

Reply:
Both are members of `Intro`: https://github.com/bitcoin/bitcoin/blob/7b701fef58f627956d597817a1f9422edd890cdc/src/qt/intro.h#L70-L71

Reply:
Done, don't know why it didn't worked last time